waspc/cli/Wasp/Cli/Command/Build.hs

```diff
@@ -31,18 +35,17 @@ build = do
       buildDirFilePath = SP.fromAbsDir buildDir
 
   doesBuildDirExist <- liftIO $ doesDirectoryExist buildDirFilePath
-  when doesBuildDirExist $
-    liftIO $ do
-      putStrLn "Clearing the content of the .wasp/build directory..."
-      removeDirectoryRecursive buildDirFilePath
-      putStrLn "Successfully cleared the contents of the .wasp/build directory.\n"
+  when doesBuildDirExist $ do
+    waspSaysC $ toWaspStartMessage "Clearing the content of the .wasp/build directory..."
+    liftIO $ removeDirectoryRecursive buildDirFilePath
+    waspSaysC $ toWaspSuccessMessage "Successfully cleared the contents of the .wasp/build directory."
 
-  liftIO $ putStrLn "Building wasp project..."
+  waspSaysC $ toWaspStartMessage "Building wasp project..."
   buildResult <- liftIO $ buildIO waspProjectDir buildDir
   case buildResult of
-    Left compileError -> throwError $ CommandError $ "Build failed: " ++ compileError
-    Right () -> liftIO $ putStrLn "Code has been successfully built! Check it out in .wasp/build directory.\n"
-  liftIO $ putStrLn alphaWarningMessage
+    Left compileError -> throwError $ CommandError $ toWaspFailureMessage "Build failed: " ++ compileError
```

Comment:
Note: The above is a contrived error screenshot where I just threw an error in `Start.hs`

Reply:
Good call, the dangling `Error: ` is not something I really like. The issue is `Wasp.Cli.Command.runCommand` could receive errors from parts of the code that do not provide a `toWaspFailureMessage` string. For example, I did not wrap:

```haskell
createNewProject :: String -> Command ()
createNewProject (all isLetter -> False) =
  throwError $ CommandError "Please use only letters for a new project's name."
createNewProject ((`elem` reservedNames) -> True) =
  throwError . CommandError $ "Please pick a project name not one of these reserved words:\n\t" ++ intercalate "\n\t" reservedNames
createNewProject name = createNewProject' (ProjectName name)
```

Do you think we should allow for any arbitrary string to be used (and hence keep Error outside), or build it into `toWaspFailureMessage` and remove it from here entirely:

```haskell
runCommand :: Command a -> IO ()
runCommand cmd = do
  errorOrResult <- runExceptT $ _runCommand cmd
  case errorOrResult of
    Left cmdError -> putStrLn $ Term.applyStyles [Term.Red] ("Error: " ++ _errorMsg cmdError)
    Right _ -> return ()
```

I think visually the latter may be better, but then it makes an implicit dependency for `throwError $ CommandError`. 🤔

Reply:
I think I will go with your suggestion, and for now just make sure all thrown command errors are failure formatted messages. It will still be colored red and the cli will die on anything it catches, so I don't think it's a big loss if we pass it an unformatted string either. Thanks 👍🏻

Reply:
Updated here: https://github.com/wasp-lang/wasp/commit/5493610a68690e90b0f2acfaa6fe8e96b1f9d605

Ended up looking like this, which I think is a good upgrade:
<img width="688" alt="Screen Shot 2022-01-04 at 11 10 18 AM" src="https://user-images.githubusercontent.com/523636/148089072-9e6b34de-a4c8-49d7-934a-b32da2c0d6f1.png">


Note: used `[Error]` as most error messages have a colon after them already.